Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Use OpenSearch image as base
+FROM opensearchproject/opensearch-dashboards:3.0.0
```

Comment:
i don't think 3.0.0 tag exists

Reply:
yes - its just for making sure that everything is aligned to the same version number (I can also replace this with a environmental var)